tests/unit/algorithms/segmentation/adapters/mmseg/datasets/pipelines/test_transforms.py

```diff
@@ -110,11 +110,11 @@ class TestNormalize:
     @pytest.mark.parametrize(
         "mean,std,to_rgb,expected",
         [
-            (1.0, 1.0, True, np.array([[[1.0, 0.0, 0.0]]], dtype=np.float32)),
-            (1.0, 1.0, False, np.array([[[-1.0, 0.0, 0.0]]], dtype=np.float32)),
+            ([1.0 for _ in range(3)], [1.0 for _ in range(3)], True, np.array([[[1.0, 0.0, -1.0]]], dtype=np.float32)),
+            ([1.0 for _ in range(3)], [1.0 for _ in range(3)], False, np.array([[[-1.0, 0.0, 1.0]]], dtype=np.float32)),
```

Comment:
Nitpick: [1.0, 1.0, 1.0] is better than [1.0 for _ in range(3)] for readability. But it's all OK.

Reply:
I also prefer [1.0, 1.0, 1.0]

Reply:
I agree with you. I'll use that format next time.